.dockerignore

```diff
@@ -0,0 +1,2 @@
+.venv
+Dockerfile
```

Comment:
empty line at the end of file + ignore more files that aren't needed to build or app (like readme, images(there is result.png in the repo), and so on)